.github/workflows/docker-build.yaml

```
@@ -43,6 +42,70 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Analyze disk usage and cleanup
+        run: |
+          echo "=== Initial disk usage ==="
+          df -h
+          echo ""
+          echo "=== Checking what large directories exist ==="
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)